extensions/amp-bodymovin-animation/0.1/amp-bodymovin-animation.js

```diff
@@ -95,6 +126,52 @@ export class AmpBodymovinAnimation extends AMP.BaseElement {
     }
     return true;
   }
+
+  play_() {
+    if (this.playingState_ == PLAYING_STATE.PLAYING) {
+      return;
+    }
+
+    const message = JSON.stringify(dict({
+      'action': 'play',
+    }));
+    this.iframe_.contentWindow./*OK*/postMessage(message, '*');
+    this.playingState_ = PLAYING_STATE.PLAYING;
+  }
+
+  pause_() {
+    if (this.playingState_ == PLAYING_STATE.PAUSED) {
+      return;
+    }
+
+    const message = JSON.stringify(dict({
+      'action': 'pause',
+    }));
+    this.iframe_.contentWindow./*OK*/postMessage(message, '*');
+    this.playingState_ = PLAYING_STATE.PAUSED;
+  }
+
+  stop_() {
+    if (this.playingState_ == PLAYING_STATE.STOPPED) {
+      return;
+    }
+
+    const message = JSON.stringify(dict({
+      'action': 'stop',
+    }));
+    this.iframe_.contentWindow./*OK*/postMessage(message, '*');
+    this.playingState_ = PLAYING_STATE.STOPPED;
+  }
+
+  seekTo_(timeVal) {
+    const message = JSON.stringify(dict({
+      'action': 'goToAndStop',
+      'value': timeVal,
```

Comment:
can we support `percent` as well? it is more useful than time ( allows scrollbound animations ). Hopefully Lottie player can give the final duration. (if loop=true, we should assume 1 loop)